app/requisiciones/routes.py

```
@@ -0,0 +1,683 @@
+    resp = make_response(pdf_data)
+    resp.headers['Content-Type'] = 'application/pdf'
+    resp.headers['Content-Disposition'] = f'attachment; filename={nombre}'
+    return resp
+
+@requisiciones_bp.route('/admin/limpiar_requisiciones_viejas') # Path is now /requisiciones/admin/limpiar_requisiciones_viejas
+@login_required
+@admin_required
+def limpiar_requisiciones_viejas_route():
+    """Limpia requisiciones finalizadas antiguas."""
+    dias = request.args.get('dias', 15, type=int)
+    eliminadas = utils.limpiar_requisiciones_viejas(dias, guardar_mensaje=True) # limpiar_requisiciones_viejas from ..utils
```

Flake8 (7.2.0)

[error] at least two spaces before inline comment

(E261)